lib/reporter.js

```diff
@@ -84,7 +84,8 @@ class CucumberReporter {
         case Cucumber.Status.PENDING:
         case Cucumber.Status.SKIPPED:
         case Cucumber.Status.AMBIGUOUS:
-            e = 'pending'
+            e = 'skipped'
```

Comment:
The point of this change is normalizing behavior of Cucumber with other frameworks.

Neither Mocha nor Jasmine doesn't emit `test:start` for pending tests. 
I tried to cancel `test:start` event for pending test, but Cucumber API is designed in a way that doesn't let us know about pending test in advance.

This is why I came up with the current solution